package.json

```diff
@@ -4047,7 +4047,7 @@
 							"null"
 						],
 						"default": null,
-						"pattern": "^((anthropic|deepseek|gemini|github|huggingface|ollama|openai|openaicompatible|openrouter|xai):([\\w.-:]+)|gitkraken|vscode)$",
+						"pattern": "^((anthropic|azure|deepseek|gemini|github|huggingface|ollama|openai|openaicompatible|openrouter|xai):([\\w.\\-:]+)|gitkraken|vscode)$",
```

Comment:
Note, the dash `-` character was not escaped here, so it was being interpreted as "any character between `.` and `:`.